app/src/androidTest/java/com/example/tanya/godeltechchallengeandroid/ui/start/MainActivityTest.kt

```diff
@@ -0,0 +1,126 @@
+package com.example.tanya.godeltechchallengeandroid.ui.start
+
+import android.app.Activity
+import android.support.v7.widget.RecyclerView
+import android.view.View
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.action.ViewActions.typeText
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.intent.rule.IntentsTestRule
+import androidx.test.espresso.matcher.RootMatchers.withDecorView
+import androidx.test.espresso.matcher.ViewMatchers.*
+import com.example.tanya.godeltechchallengeandroid.R
+import com.example.tanya.godeltechchallengeandroid.domain.entity.Word
+import com.example.tanya.godeltechchallengeandroid.ui.main.MainActivity
+import io.reactivex.Observable
+import kotlinx.android.synthetic.main.item_word_count.view.*
+import org.hamcrest.BaseMatcher
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers.*
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+
+
+class MainActivityTest {
```

Comment:
Ok... the test was written when our business logic was not implemented yet. Now it is... so we need to change the integration test to involve whole logic underlining the screen.
Now we must not mock any use case behavior! Instead we have to isolate our app from outside world and do a test with our test data. We have only one point which sees outside - it's our rest api.